src/zenml/integrations/aws/__init__.py

```diff
@@ -27,6 +27,7 @@
 AWS_SECRET_MANAGER_FLAVOR = "aws"
 AWS_CONTAINER_REGISTRY_FLAVOR = "aws"
 AWS_SAGEMAKER_STEP_OPERATOR_FLAVOR = "sagemaker"
+AWS_SAGEMAKER_ORCHESTRATOR_FLAVOR = "sagemaker"
 
 
 class AWSIntegration(Integration):
```

Comment:
Honestly, why dont we have a seperate integration for this? It seems like we are overloading the dependencies. I also had the same feeling with Vertex

Reply:
We used to, I think, but then we decided to merge them on the assumption that if someone was in the AWS ecosystem then they'd probably want it all.

Reply:
Did we decide to now keep it as it is?